### PR TITLE
Install .rpm packages and try conversion on them

### PR DESCRIPTION
**Pull request by almet**
This spawns new jobs for each platform, which:

- Checkout the dangerzone repository ;
- Install the packages from this repository ;
- Start a conversion.

Related to https://github.com/freedomofpress/dangerzone/issues/829

### CODE REVIEW

**Comment by almet** [comment]
I'm having out of space error messages:

```
* docker-archive: writing blob: adding layer with blob "sha256:17520ba7227cbffe3c9419c3f10a60a76280d6f668a46d30540912d3093627ed": processing tar file(write /usr/lib/libreoffice/program/libwriterfilterlo.so: no space left on device): exit status 1
*
```

**Comment by almet** [comment]
The CI is now passing, all seem to work well. I circumvented the space problems by moving the files rather than copying them, and removing the podman images after use.

File: .github/workflows/check-packages.yml [hunk 1]
```diff
@@ -18,4 +18,64 @@ jobs:
...
+          mkdir "./main/dist"
+          mv ./yum/dangerzone/f${{ matrix.version }}/*.rpm ./main/dist/.
+
+          # Rename files to -1.rpm, as expected by the
+          # `./dev_scripts/env.py build` command
+          pushd ./main/dist/
+          for file in dangerzone-*-2.fc*.rpm; do
+            mv "$file" "${file//-2./-1.}"
+          done
+          popd
```
Comment (apyrgio) [inline .github/workflows/check-packages.yml:56]: Can we add a comment here that this should be removed, once we're back to our usual `-1` patch level?

File: .github/workflows/check-packages.yml [hunk 2]
```diff
@@ -18,4 +18,64 @@ jobs:
...
+          ./dev_scripts/env.py --distro fedora --version ${{ matrix.version }} \
+              run dangerzone --help
+
+      - name: Clean up podman images
+        run: podman system reset -f
+
+      - name: Remove copied .rpms to free space
+        working-directory: main
+        run: |
+          rm -fr dist
```
Comment (apyrgio) [inline .github/workflows/check-packages.yml:81]: Hm, why is this needed at the very end, if the steps above have passed? The state of a run should not affect the state of other runs, even using the `matrix` strategy.

Comment (almet) [reply .github/workflows/check-packages.yml]: That's a good question. That seems to be a leftover from my previous tries where I was trying to understand why we were lacking space.

Fixed in [8281cb48736b6509749435244a2ba11b1a137845](https://github.com/freedomofpress/yum-tools-prod/pull/23/commits/8281cb48736b6509749435244a2ba11b1a137845)

File: .github/workflows/check-packages.yml (hunk 1 above)
Comment (almet) [reply .github/workflows/check-packages.yml]: Thanks, this is fixed in [`c3a74a4` (#23)](https://github.com/freedomofpress/yum-tools-prod/pull/23/commits/c3a74a4ef7503579ee1ff73ddcc2e52e4bd6b945)

**Comment by apyrgio** [comment]
Thanks for the updates Alexis, we're good to go.

**Review by apyrgio — APPROVED** [review]
